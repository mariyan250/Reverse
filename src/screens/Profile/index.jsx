// React and Style
import React from 'react';
import './index.scss';

function Profile() {
  return (
    <section className="profile">
      <h2>Profile</h2>
    </section>
  );
}

export default Profile;
