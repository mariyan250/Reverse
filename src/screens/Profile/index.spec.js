import React from 'react';
import { render } from 'tests/test-utils';
import Profile from './index';

describe('Search', () => {
  let initialState;

  beforeAll(() => {
    initialState = {};
  });

  describe('Rendering', () => {
    it('should render the component with initial state', () => {
      const { container } = render(<Profile />, initialState);
      expect(container).toMatchSnapshot();
    });
  });
});
