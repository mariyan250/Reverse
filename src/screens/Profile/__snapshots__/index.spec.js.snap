// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Rendering should render the component with initial state 1`] = `
<div>
  <section
    class="profile"
  >
    <h2>
      Profile
    </h2>
  </section>
</div>
`;
